lib/fastlane/plugin/wpmreleasetoolkit/helper/metadata_download_helper.rb

```diff
@@ -100,6 +95,33 @@ def delete_existing_metadata(target_locale)
       def get_target_file_path(locale, file_name)
         "#{@target_folder}/#{locale}/#{file_name}"
       end
+
+      private
+
+      def handle_glotpress_download(response:, locale:, is_source:)
+        case response.code
+        when '200'
+          # All good, parse the result
+          UI.success("Successfully downloaded `#{locale}`.")
+          @alternates.clear
+          loc_data = JSON.parse(response.body) rescue loc_data = nil
+          parse_data(locale, loc_data, is_source)
+          reparse_alternates(target_locale, loc_data, is_source) unless @alternates.length == 0
+        when '301'
+          # Follow the redirect
+          UI.message("Received 301 for `#{locale}`. Following redirect...")
+          download(locale, response.header['location'], is_source)
+        when '429'
+          # We got rate-limited, offer to try again
+          if UI.confirm("Retry downloading `#{locale}` after receiving 429 from the API?")
+            download(locale, response.uri, is_source)
+          else
+            UI.error("Abandoning `#{locale}` download as requested.")
```

Comment:
I decided to make this `error` so it would print red to mark the difference in behavior.

![image](https://user-images.githubusercontent.com/1218433/188542176-09a834ed-49cb-452c-99c0-ebd14f0e44a7.png)

I can see how `error`/red might be misleading though, and look like a wholesale failure. If you think that's too confusing, I can update to a `message`:

```suggestion
            UI.message("Abandoning `#{locale}` download as requested.")
```

Reply:
Given the error message is pretty clear and it's in reaction of us explicitly replying `n` in the prompt on the previous line, I vote for keeping it red with `UI.error` 👍